In [1]:
import pandas as pd

In [4]:
data = pd.read_csv('/Users/zonnid76/Downloads/top_songs.csv')

In [5]:
# To count the number of occurrences of each artist in the data
artist_counts = data['artist'].value_counts()

In [11]:
# To find the artist with the most occurences in the data
most_freqent_artist = artist_counts.idxmax()

In [10]:
most_freqent_artist

'Various Artists'

The artist that appears most in the data is "various artists", which I have confirmed is true using SQL code to double-check the data, in which "Various Artists" had 93 tracks in the dataset, followed by "Elton John" and "Taylor Swift", each of whom had 31 tracks in the dataset. 

In [13]:
# To find the average tempo in the data
average_tempo = data['tempo'].mean()

In [14]:
average_tempo

119.88951981879954

The average tempo of the tracks in the dataset is 119.88951981879954, almost 120 bpm. Using SQL to double-check this, I can confirm this is true.